articles/contributing/reporting-bugs.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Reporting bugs | Microsoft Docs
+description: Reporting bugs
+author: cgranade
+ms.author: chgranad
+ms.date: 10/12/2018
+ms.topic: article
+uid: microsoft.quantum.contributing.reporting
+---
+
+# Reporting Bugs #
+
+If you think you've found a bug in a component of the Quantum Development Kit, we would very much appreciate a report.
+After all, someone else may be struggling with the same issue as well; letting us know helps us to get to fixing it for everyone.
+The first step in reporting a bug is to start by looking through the existing issues found on each repository, as it's very possible that someone else has reported the same problem.
```

Comment:
We need a little guideline on how to figure out where the bug belongs to. 

Also, are we mentioning UserVoice at all, or are we deprecating it?

Reply:
For directly reporting issues with open source repos, my understanding was that GitHub is preferred to UserVoice, but I could have misunderstood. @cpalmer2020, do you have any suggestion as to how to clarify here? Thanks!

Reply:
Direct reporting of issues that pertain wholely to content in a GitHub repo would preferably be reported as issues on that repo.